src/towncrier/newsfragments/587.bugfix

```diff
@@ -0,0 +1 @@
+When using the default Markdown template, render a title even without a `name`
```

Comment:
I think that your release notes sound like a commit message and less as a public release announcement.

What is `name` ?

Maybe something like this? Feel free to modify my suggestion.

```suggestion
The default Markdown template now renders a title containing the release version and date, even when the `name` configuration is left empty. 
```